apps/showcase/src/assets/trainings/sdk/steps/sdk-generation/instructions.md

```diff
@@ -0,0 +1,70 @@
+There are several command lines available to generate an SDK depending on the use cases. These cases are covered in the
+<a href="https://github.com/AmadeusITGroup/otter/tree/main/packages/%40ama-sdk/schematics" target="_blank">@ama-sdk/schematics documentation</a>.
+
+### Objective
+Generate a new SDK repository using the previous specification file (Swagger Petstore) and the provided command line.
+
+### Exercise
+#### Generation of the SDK repository
+To generate a new SDK repository, you can run the following command line (by replacing `<package-name>` with `training-sdk` for example):
+
+```bash
+npm create @ama-sdk typescript <package-name>
+```
+
+You will receive a prompt asking you to specify the project name. For example, you can set the project name to `training-project`.
+
+```bash
+? Project name (NPM package scope, package.json name will be @{projectName}/{packageName})? training-project
+```
+
+After setting the project name, you will see the shell of your SDK repository being generated with a structure that looks something like the following:
+```bash
+src
+├── api
+├── fixtures
+├── helpers
+├── models
+│   ├── base
+│   ├── core
+│   └── custom
+└── spec
```

Comment:
```suggestion
```mermaid
graph LR
    src --> api
    src --> fixtures
    src --> helpers
    src --> models
    src --> spec
    
    models --> base
    models --> core
    models --> custom
```

(or less fancy but not flagged `bash`)

Reply:
I removed the `bash` flag (I put it there to add some color but keeping it simple is fine too :))